caf-audit-service-container/src/integration-test/java/com/hpe/caf/services/audit/api/AuditConnectionHelper.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.audit.api;
+
+import com.hpe.caf.api.ConfigurationException;
+import com.hpe.caf.api.ConfigurationSource;
+import com.hpe.caf.auditing.AuditConnection;
+import com.hpe.caf.auditing.AuditConnectionFactory;
+import com.hpe.caf.auditing.elastic.ElasticAuditConfiguration;
+import com.hpe.caf.auditing.webserviceclient.WebserviceClientAuditConfiguration;
+
+public class AuditConnectionHelper
+{
+
+    public static AuditConnection getElasticAuditConnection(String esHostAndPorts, String esClusterName)
+        throws ConfigurationException
+    {
+
+        return AuditConnectionFactory.createConnection(new ConfigurationSource()
+        {
+
+            @Override
+            public <T> T getConfiguration(Class<T> aClass) throws ConfigurationException
+            {
+                ElasticAuditConfiguration elasticAuditConfiguration = new ElasticAuditConfiguration();
+                elasticAuditConfiguration.setHostAndPortValues(esHostAndPorts);
+                elasticAuditConfiguration.setClusterName(esClusterName);
+                return (T) elasticAuditConfiguration;
+            }
+        });
+    }
+
+    public static AuditConnection getWebserviceAuditConnection(String wsHostAndPort)
```

Comment:
I will revisit the javadoc of this in my branch that adds support for NO_PROXY, HTTP_PROXY and HTTPS_PROXY settings as method parameter names have changed here